qt/aqt/main.py

```diff
@@ -402,15 +404,14 @@ def doOpen(path: str) -> None:
         )
 
     def _openBackup(self, path: str) -> None:
-        def on_done(success: bool) -> None:
-            if success:
-                self.onOpenProfile(callback=lambda: self.col.mod_schema(check=False))
-
-        import aqt.importing
-
         self.restoring_backup = True
         showInfo(tr.qt_misc_automatic_syncing_and_backups_have_been())
-        aqt.importing.replace_with_apkg(self, path, on_done)
+
+        import_collection_package_op(self, path).success(
+            lambda _: self.onOpenProfile(
+                callback=lambda: self.col.mod_schema(check=False)
```

Comment:
I think we can safely remove the mod_schema() line now, due to 6735b23e5bd3257c1c36d48f4068b991a3c24556